package.json

```diff
@@ -79,7 +79,7 @@
     "flow:stop": "flow stop",
     "build:lib": "node -e \"require('ncp').ncp('./src', './lib')\" && BABEL_ENV=production babel -d lib src",
     "build:browser": "webpack --config webpack.config.js",
-    "build": "npm run build:lib && npm run build:node",
+    "build": "npm run build:lib && npm run build:browser",
```

Comment:
Oh, I just took a closer look at the code. Is `build:lib` really `build:node`? That would make things clearer I think if that is the case. Then you could rename and leave the code as is. 

`npm run build:node && npm run build:browser` makes it obvious what's going on.

If `build:node` is another separate step and both `build:node` and `build:browser` require `build:lib`, I would chnage it this way:

```json
"prepublish": "npm run build:lib",
"publish": "npm run build:node && npm run build:browser",
```